src/components/organisms/pupil/OakPupilJourneyHeader/OakPupilJourneyHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { OakFlex, OakHeading, OakIconProps } from "../../../atoms";
+import {
+  OakBulletList,
+  OakBulletListProps,
+  OakHandDrawnCardWithIcon,
+} from "../../../molecules";
+
+export type OakPupilJourneyHeaderProps = {
+  title: string;
+  iconName: OakIconProps["iconName"];
+  alt?: OakIconProps["alt"];
+  iconBackground?: "primary" | "secondary";
+} & OakBulletListProps;
```

Comment:
given this is a specialised component , I think we could rename listItems to something more specific. Eg. breadcrumbs ?